tests/page-layout-both_spread/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>If a reflowable spine item has a page-spread-* value, it must be honored.</dc:description>
+    <dc:identifier id="pub-id">page-layout-both_spread</dc:identifier>
```

Comment:
Same question as previous test: Why is this called page-layout-both(_spread)? I don't see anything about page-layout in the package, only layout-reflowable and page-spread-left. Can you come up with a better name (both title and ID/filenames)?

Reply:
This is to test spread behaviour for a page-layout-both title, so while the name for both tests is bad, it's at least consistent?